extensions/xdebug/lib/XDebugExtension.php

```diff
@@ -44,10 +44,14 @@ public function configure(OptionsResolver $resolver): void
 
     public function load(Container $container): void
     {
+        $container->register(XDebugUtil::class, function (Container $container) {
+            return new XDebugUtil(phpversion('xdebug'), ini_get('xdebug.use_compression'));
```

Comment:
prob need to cast `ini_get` to a bool?

Reply:
probably need to handle these values properly - what does `phpversion` return if `xdebug` is not set? `init_get` will return `false` if the init settings does not exist, but a string otherwise... does it return `1` and `0` ?